src/cljs/main/broadfcui/page/workspace/monitor/submission_details.cljs

```diff
@@ -27,78 +28,93 @@
-                        :href (nav/get-link :workspace-workflow
-                                            (:workspace-id props)
-                                            (:submission-id props)
-                                            id)})))}
-                 {:header "Data Entity" :starting-width 200
-                  :as-text
-                  (fn [entity]
-                    (str (:entityName entity) " (" (:entityType entity) ")"))
-                  :sort-by :text}
-                 {:header "Last Changed" :starting-width 280 :as-text moncommon/render-date}
-                 {:header "Status" :starting-width 120
-                  :content-renderer (fn [status]
-                                      [:div {}
-                                       (moncommon/icon-for-wf-status status)
-                                       status])}
-                 {:header "Messages" :starting-width 300
-                  :content-renderer (fn [message-list]
-                                      [:div {}
-                                       (map (fn [message]
-                                              [:div {} message])
-                                            message-list)])}
-                 {:header "Workflow ID" :starting-width 300
-                  :as-text
-                  (fn [workflow] (:workflowId workflow))
-                  :sort-by :text
-                  :content-renderer
-                  (fn [workflow]
-                    (let [{:keys [submission-id bucketName]} props
-                          inputs (second (second (first (:inputResolutions workflow))))
-                          input-names (string/split inputs ".")
-                          workflow-name (first input-names)
-                          workflowId (:workflowId workflow)]
-                      (style/create-link {:text workflowId
-                                          :target "_blank"
-                                          :style {:color "-webkit-link" :textDecoration "underline"}
-                                          :href (str moncommon/google-cloud-context bucketName "/" submission-id "/"
-                                                     workflow-name "/" workflowId "/")})))}]
-       :filter-groups
-       (vec (cons {:text "All" :pred (constantly true)}
-                  (map (fn [status] {:text status :pred #(= status (:status %))})
-                       moncommon/wf-all-statuses)))
-       :data (:workflows props)
-       :->row (fn [row]
-                [(:workflowId row)
-                 (:workflowEntity row)
-                 (:statusLastChangedDate row)
-                 (:status row)
-                 (:messages row)
-                 row])}])
+      {:ref "table"
+       :data (:filtered-data @state)
+       :body
+       {:empty-message "No Workflows"
+        :style table-style/table-heavy
+        :behavior {:fixed-column-count 1}
+        :columns [
+                   {:id "view"
```

Comment:
```clojure
:columns [{:id "view"
```
And reindent from there. In general, an open bracket should never be the last thing on a line, and a close bracket should never be the first thing on a line.